sdks/python/apache_beam/ml/inference/api.py

```diff
@@ -52,8 +52,12 @@ class RunInference(beam.PTransform):
 
   TODO(BEAM-14046): Add and link to help documentation
   """
-  def __init__(self, model_loader: base.ModelLoader):
-    self._model_loader = model_loader
+  def __init__(
+      self, model_loader: Union[base.ModelLoader, Callable], return_row=False):
+    self._model_loader = model_loader() if isinstance(
```

Comment:
I think this is a case where we'll want to provide a wrapping transform rather than do a type check (which would also force model loaders to never implement callable). Mostly that is because I think we well almost always want to provide arguments to the model loader, and being able to pass `PythonCallable('path.to.ModelType', arg0, arg1, ...)` will be more convenient (and less error prone) than trying to inline the arguments in the PythonCallable string itself.

Reply:
Adding `create` method so that we could separate the class constructor and the arguments. The low-level API from the Java SDK would look like:
```
PythonExternalTransform.from("apache_beam.ml.inference.api.RunInference.create")
    .withKwarg("model_loader_provider", PythonCallableSource.of(
        "from apache_beam.ml.inference.sklearn_inference import SklearnModelLoader\nSklearnModelLoader"))
    .withKwarg("model_uri", "/tmp/staged/sklearn_model")
```